src/Serilog.Sinks.Udp/Sinks/Udp/Private/UdpSink.cs

```diff
@@ -89,8 +151,15 @@ protected override async Task EmitBatchAsync(IEnumerable<LogEvent> events)
                                 .Trim()
                                 .ToCharArray());
 
+                        if (this.remoteEndPoint == null)
+                        {
+                            await client
+                                .SendAsync(buffer, buffer.Length, this.remoteHostname, this.remotePort)
+                                .ConfigureAwait(false);
+                        }
+
                         await client
```

Comment:
Shouldn't this be placed in an `else` clause?

Reply:
Yeah, my bad